fundingmgr.go

```diff
@@ -87,14 +87,7 @@ type fundingMgr struct {
 	// node doesn't already have channels with.
 	newNodesOnly bool
 
-	// pendingOpenChannels is a channel through which we'll receive
-	// notifications for pending open channels resulting from a successful
-	// batch.
-	pendingOpenChannels <-chan *lnrpc.ChannelEventUpdate_PendingOpenChannel
```

Comment:
Is there a reason this can't continue to live within the funding manager since that's where we open channels?

Reply:
I wasn't sure how exactly to do it, this is mostly needed for #184. But I think it does make sense to make the funding manager the owner over the update chan :+1: